minos/microservice/cli.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import (
+    Path,
+)
+from typing import (
+    Optional,
+)
+
+import typer
+
+from minos.common import (
+    MinosConfig,
+)
+from minos.microservice import (
```

Comment:
Hi @vladyslav-fenchak, could you change this import to a relative one? 
```suggestion
from .entrypoint import (
```